cmd/dump/src/lib.rs

```diff
@@ -426,6 +444,52 @@ fn emulate_dump(core: &mut dyn Core, base: u32, total: u32) -> Result<()> {
     Ok(())
 }
 
+fn emulate_task_dump(
```

Comment:
I'm confused by this function name – it looks like it's just setting up the dump segments, rather than actually doing the dump, in contrast with `emulate_dump`.